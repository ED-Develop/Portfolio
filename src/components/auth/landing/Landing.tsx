import React, {FC} from 'react';
import style from '../Auth.module.scss';
import {Banner} from '../banner/Banner';
import {Layout} from 'antd';
import {LoginAside} from '../asside/Asside';

export const Landing: FC = ({children}) => {
    return (
        <Layout className={style.container}>
            <Banner/>
            <LoginAside>{children}</LoginAside>
        </Layout>
    )
};


