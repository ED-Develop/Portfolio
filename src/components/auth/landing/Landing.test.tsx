import React from 'react';
import {render} from '@testing-library/react';
import {Landing} from './Landing';
import {BrowserRouter} from 'react-router-dom';

describe('Landing component', () => {
    test('Should renders correctly', () => {
        const {container} = render(
            <BrowserRouter>
                <Landing/>
            </BrowserRouter>
        );

        expect(container).toMatchSnapshot();
    });
})