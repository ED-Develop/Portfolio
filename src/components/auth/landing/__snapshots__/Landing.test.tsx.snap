// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component Should renders correctly 1`] = `
<div>
  <section
    class="ant-layout ant-layout-has-sider container"
  >
    <main
      class="ant-layout-content landing__banner"
    >
      <div
        class="logo"
      >
        <img
          alt="logo"
          src="logo-dark.png"
        />
      </div>
      <div
        class="title"
      >
        <h1>
          Make your life more interesting with Socialite Social Network
        </h1>
      </div>
      <ul
        class="nav"
      >
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            About
          </a>
        </li>
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            Contact
          </a>
        </li>
      </ul>
    </main>
    <aside
      class="sider ant-layout-sider ant-layout-sider-dark"
      style="flex: 0 0 33%; max-width: 33%; min-width: 33%; width: 33%;"
    >
      <div
        class="ant-layout-sider-children"
      >
        <div
          class="landing__sider"
        />
      </div>
    </aside>
  </section>
</div>
`;
